detox/e2e/test/smoke_test/edit_channel.e2e.js

```diff
@@ -11,42 +11,166 @@ import {
+} from '@support/server_api';
 
 describe('Edit Channel', () => {
+    const {
+        editChannelScreen,
+        headerInput,
+        nameInput,
+        purposeInput,
+        saveButton,
+    } = EditChannelScreen;
+    const {
+        addMembersAction,
+        archiveAction,
+        channelDisplayName,
+        channelInfoScrollView,
+        channelPurpose,
+        editChannelAction,
+        favoritePreferenceAction,
+        ignoreMentionsPreferenceAction,
+        manageMembersAction,
+        mutePreferenceAction,
+        notificationPreferenceAction,
+        leaveAction,
+        pinnedMessagesAction,
+    } = ChannelInfoScreen;
+    const {
+        getUserAtIndex,
+        searchInput,
+        startButton,
+    } = MoreDirectMessagesScreen;
+    const {
+        goToChannel,
+        openMainSidebar,
+    } = ChannelScreen;
+    let testUser;
+    let testOtherUser1;
+    let testOtherUser2;
+    let testChannel;
+
     beforeAll(async () => {
-        const {user} = await Setup.apiInit();
+        const {channel, team, user} = await Setup.apiInit();
+        testUser = user;
+        testChannel = channel;
+
+        ({user: testOtherUser1} = await User.apiCreateUser());
+        await Team.apiAddUserToTeam(testOtherUser1.id, team.id);
+
+        ({user: testOtherUser2} = await User.apiCreateUser());
+        await Team.apiAddUserToTeam(testOtherUser2.id, team.id);
 
         // # Open channel screen
-        await ChannelScreen.open(user);
+        await ChannelScreen.open(testUser);
     });
 
     afterAll(async () => {
         await ChannelScreen.logout();
     });
 
-    it('MM-T3199 should be able to edit public channel', async () => {
-        const {
-            editChannelScreen,
-            headerInput,
-            nameInput,
-            purposeInput,
-            saveButton,
-        } = EditChannelScreen;
-        const {
-            channelDisplayName,
-            channelPurpose,
-        } = ChannelInfoScreen;
+    it('MM-T3190 should be able to display channel info', async () => {
+        // # Open channel info screen
+        await goToChannel(testChannel.display_name);
+        await ChannelInfoScreen.open();
+
+        // * Verify channel info fields are visible
+        await expect(channelDisplayName).toHaveText(testChannel.display_name);
+        await expect(channelPurpose).toHaveText(`Channel purpose: ${testChannel.display_name.toLowerCase()}`);
+        await expect(element(by.text(`Channel header: ${testChannel.display_name.toLowerCase()}`))).toBeVisible();
+        await expect(favoritePreferenceAction).toBeVisible();
+        await expect(mutePreferenceAction).toBeVisible();
+        await expect(ignoreMentionsPreferenceAction).toBeVisible();
+        await expect(notificationPreferenceAction).toBeVisible();
+        await expect(pinnedMessagesAction).toBeVisible();
+        await expect(manageMembersAction).toBeVisible();
+        await channelInfoScrollView.scrollTo('bottom');
```

Comment:
This scroll action seemed to not work on Android (at least on my machine).

Reply:
@saturninoabril I think this is a general Android emulator issue. I experienced it once a while back and was scratching my head, the I just decided to wipe data on the emulator and restart my computer. May be try the same and see if that helps.

Reply:
@saturninoabril I just merged master so you might want to pull the latest.